src/js/disability-benefits/526EZ/config/form.js

```diff
@@ -120,110 +128,63 @@ const formConfig = {
             type: 'object',
+            required: ['fullName', 'gender', 'dateOfBirth'],
             properties: {
-              'view:suicidal': {
-                type: 'boolean'
+              fullName,
+              socialSecurityNumber: {
+                type: 'string'
               },
-              'view:homeless': {
-                type: 'boolean'
+              vaFileNumber: {
+                type: 'string',
+                pattern: '^[cC]{0,1}\\d{7,9}$'
               },
-              'view:extremeFinancialHardship': {
-                type: 'boolean'
+              gender: {
+                type: 'string',
+                'enum': ['F', 'M']
               },
-              'view:blindOrSightImpaired': {
-                type: 'boolean'
-              }
+              dateOfBirth: date
             }
           }
         },
-        militaryHistory: {
-          title: 'Military service history',
-          path: 'review-veteran-details/military-service-history',
-          depends: formData => formData.prefilled,
-          'ui:description': 'things',
-          initialData,
-          uiSchema: {
-            servicePeriods: {
-              'ui:title': 'Military service history',
-              'ui:description':
-                'This is the service history we have on file for you. If you need to update your service history, you can edit or add another service period.',
-              'ui:options': {
-                itemName: 'Service Period',
-                viewField: ServicePeriodView,
-                reviewMode: true
-              },
-              items: {
-                serviceBranch: {
-                  'ui:title': 'Branch of service'
-                },
-                dateRange: dateRangeUI(
-                  'Service start date',
-                  'Service end date',
-                  'End of service must be after start of service'
-                ),
-                dischargeType: {
-                  'ui:title': 'Character of discharge',
-                  'ui:options': {
-                    labels: {
-                      honorable: 'Honorable',
-                      general: 'General',
-                      other: 'Other Than Honorable',
-                      'bad-conduct': 'Bad Conduct',
-                      dishonorable: 'Dishonorable',
-                      undesirable: 'Undesirable'
-                    }
-                  }
-                }
-              }
-            }
-          },
-          schema: {
-            type: 'object',
-            properties: {
-              servicePeriods
-            }
-          }
-        }
-      }
-    },
-    veteranDetails: {
-      title: 'Veteran Details',
-      pages: {
-        veteranInformation: createUnverifiedVeteranInfoPage(fullSchema526EZ),
         specialCircumstances: {
           title: 'Special Circumstances',
           path: 'special-circumstances',
+          depends: formData => formData.prefilled,
```

Comment:
Will these pages always be displayed now?

Reply:
Yeah, exactly. We'll have just one set of pages that are always shown.

Reply:
Ok, great, should we remove the depends?